qa-ldap-process.php

```diff
@@ -69,18 +69,17 @@ function isEmpty($attr) {
           qa_redirect('login');
           exit();
         }
+        // REMY BLOM: DO WHAT THE SCRIPT IS SUPPOSED TO DO, DON'T SET COOKIES?!?!?!
```

Comment:
Please remove this comment as well since it won't make any sense after this PR is merged.